File: 8_getting_inputs/input.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

// getting an input from the console
// in Golang

func main() {
	reader := bufio.NewReader(os.Stdin) // reads from stdin (input)
	fmt.Print("What is your name?")     // prompt msg

	line, _ := reader.ReadString('\n') // reads string till new line occur

	// I used print format and I removed new line separator from the input
	fmt.Printf("Hello %s :) \n", strings.Replace(line, "\n", "", -1))
}

File: 8_getting_inputs/input.js
// getting an input from the console
// in JavaScript (using NodeJS)

const readline = require("readline")

// Reader Object to read from stdin (input) and write to (stdout)
let reader = readline.createInterface(process.stdin, process.stdout);

reader.question("What is your name?", (answer)=> {
    reader.write(`Hello ${answer} :) \n`); // Writes to stdout (output)
    reader.close(); // Closes the reader
});
